src/qt_gui/translations/pl_PL.ts

```diff
@@ -16,322 +16,322 @@
         <message>
             <location filename="../about_dialog.ui" line="78"/>
             <source>shadPS4 is an experimental open-source emulator for the PlayStation 4.</source>
-            <translation>shadPS4 is an experimental open-source emulator for the PlayStation 4.</translation>
+            <translation>shadPS4 to eksperymentalny otwarto-źródłowy emulator PlayStation 4.</translation>
         </message>
         <message>
             <location filename="../about_dialog.ui" line="99"/>
             <source>This software should not be used to play games you have not legally obtained.</source>
-            <translation>This software should not be used to play games you have not legally obtained.</translation>
+            <translation>Nie powinien być używany w celu ogrywania nielegalnych kopii gier.</translation>
```

Comment:
Nit: Even thought entire sentence conveys appropriate message, I wonder whether used verb "ogrywać" is somewhat informal for legal disclaimer?

Reply:
I'm backing it up as I had exactly the same feeling about it.